### ISSUE TITLE
Please don't upgrade image ethpandaops/ethereum-genesis-generator automatically

### ISSUE THREAD

**Issue by shaneing** [kurtosis-tech/kurtosis#1592]
### Background & motivation

Suddenly, I found my 1-click script could not work well due to the upgrade of ethpandaops/ethereum-genesis-generator from 2.0.0 to 2.0.4. It causes a problem of getting an empty account list.

### Desired behaviour

Don't upgrade ethpandaops/ethereum-genesis-generator automatically.

### How important is this to you?

Nice to have; this feature would make using Kurtosis more enjoyable.

### What area of the product does this pertain to?

CLI: the Command Line Interface

**Comment by shaneing** [comment]
Sorry, the problem is caused by [ethereum-package](https://github.com/kurtosis-tech/ethereum-package). Can we specify the version when run the eth-network?

**Comment by shaneing** [comment]
> Sorry, the problem is caused by [ethereum-package](https://github.com/kurtosis-tech/ethereum-package). Can we specify the version when run the eth-network?

It works well by specifying the version, github.com/kurtosis-tech/ethereum-package@0.6.0.

**Comment by leeederek** [comment]
Hey @shaneing - thanks for filing this issue. I'm glad that you got unblocked on your own. In the future, we're generally more responsive in our [Discord Server](https://discord.gg/Sr8PkkBY7d) in case you need help again. 

I'm not sure if this is how you did it, but you can also pin a specific version, fork, or commit in the `kurtosis.yml` file using the `replace` primitive: https://docs.kurtosis.com/concepts-reference/kurtosis-yml/#replace
